test-requirements.ini

```diff
@@ -0,0 +1,2 @@
+geerlingguy.java
+ernestas-poskus.zookeeper
```

Comment:
Since 'https://github.com/AnsibleShipyard/ansible-zookeeper' is not published in Ansible galaxy had to fork it & add it myself. Would you mind publishing original role ?